<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgProperty</class>
 <widget class="QDialog" name="DlgProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="8" column="1">
    <widget class="QLineEdit" name="txtNameSpace">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>#images</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="txtURL">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QCheckBox" name="chkGenerateBgImage">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>生成单独的background-image选择器</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QGroupBox" name="grpMain">
     <property name="title">
      <string>元素CSS模板</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="txtTemplate">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>/*[%t|%r] (%w,%h)+(%x,%y) */
%ns%i {
	width: %wpx;
	height: %hpx;
	background-position: -%xpx -%ypx;
}
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblShow4">
        <property name="text">
         <string>可用变量：
%i - 元件选择器
%ns - 命名空间
%r - 元件名称
%t - 元件类型
%x - X位置
%y - Y位置
%w - 宽度
%h - 高度</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="lblShow3">
     <property name="text">
      <string>背景图片选择器</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="lblShow2">
     <property name="text">
      <string>图片url</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblShow1">
     <property name="text">
      <string>命名空间</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblShow6">
     <property name="text">
      <string>图像高度</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLineEdit" name="txtBgSelector">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>.image</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblShow5">
     <property name="text">
      <string>图像宽度</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="btnsMain">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Reset|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="txtWidth">
     <property name="minimum">
      <number>12</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btnCutSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Not Implement</string>
     </property>
     <property name="icon">
      <iconset theme="zoom-fit-best"/>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QSpinBox" name="txtHeight">
     <property name="minimum">
      <number>12</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>裁剪画布</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnsMain</sender>
   <signal>accepted()</signal>
   <receiver>DlgProperty</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnsMain</sender>
   <signal>rejected()</signal>
   <receiver>DlgProperty</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAnyChange()</slot>
 </slots>
</ui>
